scp.py

```diff
@@ -119,6 +120,17 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self.close()
 
+    def _progress_tracker(self, method, basename, size, file_pos, peername=""):
```

Comment:
Can you remove the default value for `peername`? I don't think `""` makes sense (it's not a tuple). That argument is always supplied anyway.